packages/roosterjs-editor-api/lib/format/toggleCodeBlock.ts

```diff
@@ -0,0 +1,36 @@
+import { Editor } from 'roosterjs-editor-core';
+import { getTagOfNode, unwrap, wrap } from 'roosterjs-editor-dom';
+import toggleTagCore from './toggleTagCore';
+
+const PRE_TAG = 'pre';
+const CODE_TAG = 'code';
+const CODE_NODE_TAG = 'CODE';
+const defaultStyler = (_: HTMLElement) => {};
```

Comment:
Better move this into toggleTagCore() so we don't need always pass it.

Reply:
Or check null in toggleTagCore() and dothing for null case

Reply:
I'll move it, generally prefer null functions over if checks